index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <![endif]-->
+<html lang="fr">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Portfolio: setup and mobile version skeleton</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <nav class="navbar">
+            <div class="logo">My Logo</div>
+            <ul class="nav-links">
+                <li><label id="checkbox_toggle" class="hamburger">&#9776;</label></li>
+            </ul>
+        </nav>
+    </header>
+    <section>
+        <div class="body-bar">
+            <div class="align-center-text">
+                <h2 class="header-text">I'm Abol<br>Glad to see <br>you!</h2>
```

Comment:
- [ ] Please wrap this section inside of a `h1` tag as instructed earlier and not `h2` this is because it is the main title of the page and also to follow semantics.

Reply:
@Mre55